enumerable_method.rb

```diff
@@ -205,4 +335,16 @@ def my_map_proc(&block)
     end
     newarray
   end
```

Comment:
The name of the method is `multiply_els` and is not an enumerable method. So it should be outside of the module.

Reply:
I did change it